src/openklant/utils/validators.py

```diff
@@ -39,7 +39,15 @@ def validate_charfield_entry(value, allow_apostrophe=False):
 
 
 validate_phone_number = RegexValidator(
-    regex="(0[8-9]00[0-9]{4,7})|(0[1-9][0-9]{8})|(\\+[0-9]{9,20}|1400|140[0-9]{2,3})",
+    regex=r"^("
+    r"0[8-9]00[0-9]{4,8}"  # starting with 08 or 09, followed by 00 and then 4 to 8 digits
```

Comment:
Thanks for splitting this up, it makes it easier to understand with the added comments :+1: